assets/scss/core/_settings.scss

```diff
@@ -5,17 +5,14 @@
 ////
 /// Settings ///
 ///
-$ds-background: $ds-color-ghostwhite !default;
-$ds-background-bright: $ds-background !default;
-$ds-wrapper-min-width: torem(1200px) !default;
-$ds-breakpoints: (
-  'desktop-fhd': (max-width: 1920px),
-  'desktop': (max-width: 1366px),
-  'ipad': (min-width: 1023px, max-width: 1024px),
-  'isdesktop': (min-width: 1024px),
-  'ismobile': (max-width: 1024px),
-  'iphone': (max-width: 980px)
+$dp-background: $dp-color-ghostwhite !default;
+$dp-wrapper-min-width: 1200px !default;
```

Comment:
this values talked with Mariana?

Reply:
We'll talk to Mariana on her return

Reply:
Well, we could take a temporal decision, @MarianaMS can change it later.